Runtime/Plugins/GLTFSerialization/Schema/GLTFProperty.cs

```diff
@@ -37,7 +37,8 @@ public static IReadOnlyList<string> RegisteredExtensions
 			{ KHR_draco_mesh_compression_Factory.EXTENSION_NAME, new KHR_draco_mesh_compression_Factory() },
 			{ KHR_texture_basisu_Factory.EXTENSION_NAME, new KHR_texture_basisu_Factory() },
 			{ EXT_meshopt_compression_Factory.EXTENSION_NAME, new EXT_meshopt_compression_Factory() },
-			{ EXT_mesh_gpu_instancing_Factory.EXTENSION_NAME, new EXT_mesh_gpu_instancing_Factory() }
+			{ EXT_mesh_gpu_instancing_Factory.EXTENSION_NAME, new EXT_mesh_gpu_instancing_Factory() },
+			{ KHR_animation_pointerExtensionFactory.EXTENSION_NAME, new KHR_animation_pointerExtensionFactory() }
 		};
```

Comment:
add comma at end of entry so future changes don't have to change the line again

Reply:
will be changed in the Dispersion branch with commit 5de6b1d06efda04df7635b3440a7f9309e291529